foundation/foundation-mda/src/main/resources/targets.json

```diff
@@ -2413,6 +2413,22 @@
     "metadataContext": "targeted",
     "overwritable": false
   },
+  {
+    "name": "baseHiveProperties",
+    "templateName": "templates/deployment/hive-metastore-service/hive.properties.vm",
+    "outputFile": "configurations/base/hive.properties",
+    "generator": "com.boozallen.aiops.mda.generator.ModelAgnosticResourcesGenerator",
+    "metadataContext": "targeted",
+    "overwritable": false
+  },
+  {
+    "name": "envHiveProperties",
+    "templateName": "templates/deployment/hive-metastore-service/hive.properties.vm",
+    "outputFile": "configurations/env/hive.properties",
+    "generator": "com.boozallen.aiops.mda.generator.ModelAgnosticResourcesGenerator",
+    "metadataContext": "targeted",
+    "overwritable": false
+  },
```

Comment:
Think we can just generate the base properties file. No point in having a env specific one that sets them to the exact same value. If the downstream uses wants them to be env specific, they can add that file.

Reply:
Yup I tried without env but seems like I am getting error from config store when I stood up argoCD.
That was why I added env as placeholder and consumer can update freely.
Here are screenshot of error message without env.
<img width="1393" alt="Screenshot 2024-12-06 at 11 24 11 AM" src="https://github.com/user-attachments/assets/6bc7d15e-d499-413d-afee-af2222321053">
<img width="1100" alt="Screenshot 2024-12-06 at 11 24 24 AM" src="https://github.com/user-attachments/assets/355e7077-5a61-44b9-9bf4-1f8d1e262883">

Reply:
Does it work if you make the environment file blank? Think it could be useful make a generic envProperties template that just includes something like:
```
# Add environment specific overrides here. See the documentation for more details:
# https://boozallen.github.io/aissemble/aissemble/current/configuration-store.html#_environment_specific_overrides
```
That way as we build out more property files for the config store, we can reuse the generic one for our env property files.

Reply:
yup it is working with empty properties file in env =) . Ill update to make this one empty.